### PR TITLE
Spdif Transmit refresh

### PR DESCRIPTION
**Pull request by TDW89**
Update and refresh of SpdifTransmit.xc along with some tidy up of externally referenced header files (should now be using the api file)

### CODE REVIEW

**Review by xross — CHANGES_REQUESTED** [review]

**Comment by xross** [comment]
SpdifTransmit.xc has inconsistent indentation - sometimes two spaces, sometimes four

File: lib_spdif/api/spdif.h [hunk 1]
```diff
@@ -44,6 +44,39 @@
...
+ * \param c               channel to output samples to
+ *
+ * \param initial_divider initial divide for initial estimate of sample rate
+ *                        For a 100Mhz reference clock, use an initial divider
+ *                        of 1 for 192000, 2 for 96000/88200, and 4 for 48000/44100.
+ *
+ * \param clk             clock block sourced from the 100 MHz reference clock.
+ *
+ **/
+void SpdifReceive(in buffered port:4 p, streaming chanend c, int initial_divider, clock clk);
```
[the remark this answers is not in the document]
Comment (xross) [reply lib_spdif/api/spdif.h:78]: This one still needs addressing

**Review by xross — CHANGES_REQUESTED** [review]

File: lib_spdif/api/spdif.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (TDW89) [reply lib_spdif/api/spdif.h]: should be sorted

File: lib_spdif/src/SpdifTransmit.xc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2011-2022 XMOS LIMITED.
+// Copyright 2011-2023 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 
 /**
```
[the remark this answers is not in the document]
Comment (TDW89) [reply lib_spdif/src/SpdifTransmit.xc:4]: fixed

**Comment by xross** [comment]
Looks good, I quickly scanned the documentation and didn't see any updated that were required but would be good to get a second set of eyes to look over it.

**Review by xross — APPROVED** [review]

**Pull request merged**